lib/sycamore/sycamore/tests/unit/transforms/test_detr_partitioner.py

```diff
@@ -86,3 +87,69 @@ def test_table_extraction_order(self):
             use_cache=False,
         )
         assert len(d) == 1
+
+    def test_pdfminer_object_type(self):
+        filename = str(TEST_DIR / "resources/data/pdfs/Ray_page11.pdf")
+        lines_extractor = TextExtractor("pdfminer", object_type="lines")
+        pages = PdfMinerExtractor.pdf_to_pages(file_name=filename)
+
+        lines_elements = []
+        for i, p in enumerate(pages):
+            assert i == 0
+            lines_elements.extend(lines_extractor.extract_page(p))
+
+        objects_extractor = TextExtractor("pdfminer", object_type="boxes")
+        pages = PdfMinerExtractor.pdf_to_pages(file_name=filename)
+
+        objects_elements = []
+        for i, p in enumerate(pages):
+            assert i == 0
+            objects_elements.extend(objects_extractor.extract_page(p))
+
+        # Note: It's possible that for some documents these values would be equal, but
+        # I was worried that if I used <= I might silently mask an issue where we weren't
+        # actually honoring the object_type parameter. For this doc the values are quite
+        # different
+        assert len(objects_elements) < len(lines_elements)
+        print(f"objects_elements {len(objects_elements)}, lines_elements {len(lines_elements)}")
+
+        lines_text = "".join([e.text_representation for e in lines_elements if e.text_representation is not None])
+        objects_text = "".join([e.text_representation for e in objects_elements if e.text_representation is not None])
+
+        # I was very surprised that this equality succeeded. I'm not sure in general we can expect
+        # exact text equality. I imagine in some cases the order might be different, but in this case
+        # they match, so I'm asserting here so we can catch regressions.
+        assert lines_text == objects_text
```

Comment:
Not sure what you mean by having empty lines_elements. Since each box is a grouping of lines, it is always the case that len(lines) >= len(objects) -- I should probably have called it boxes instead of objects here. I made the inequality strict to ensure that we are actually doing something different. The user should not notice. By the time it gets to them, either lines or boxes will be merged into elements.

Reply:
Should have phrased it differently, I mean that since `lines_text == objects_text` and len(lines_elements) > len(objects_elements), we would have some elements that don't have any text in them. Shouldn't be a real performance issue but wanted to flag it for the customer side.

Reply:
I think the common case is that all the elements have text, just that the lines_elements have less text. But again, these are the elements that come directly out of pdfminer. They get grouped into larger elements in `_suplement_text` before they get returned to the users. So users shouldn't see a difference.